engine/table/src/test/java/io/deephaven/engine/util/TimeTableTest.java

```diff
@@ -43,7 +44,12 @@ protected void setUp() throws Exception {
         super.setUp();
 
         now = new MutableLong(0);
-        timeProvider = () -> new DateTime(now.longValue());
+        clock = new ClockNanoBase() {
```

Comment:
We do this a lot. Maybe we want one based on a `MutableLong`?

Reply:
Elevated TestClock.